<?xml version="1.0"?>
<launch>
    
  <arg name="camera_ns" default="gripper_cam" />
  <arg name="image_topic" default="image_raw" />   

  <node pkg="hector_motion_detection" type="motion_detection_node" name="motion_detection" output="screen" respawn="true">
    <param name="enabled" value="false" />
    <rosparam file="$(find hector_motion_detection)/config/moving_dots.yaml" command="load" subst_value="True"/>
    
    <remap from="~detection/image_detection" to="/detection/visual_detection" />
    <remap from="~image" to="$(arg camera_ns)/$(arg image_topic)" />
    <remap from="~enabled_status" to="/startup_check/motion_detection/enabled_status" />
    <remap from="~enabled" to="/startup_check/enabled" />
  </node>

</launch>
